templates/zerver/config_error.html

```diff
@@ -10,15 +10,15 @@
                 <div class="errorcontent">
                     <h1 class="lead">Configuration error</h1>
                     <br/>
-                    {% if ldap_error_realm_is_none %}
+                    {% if error_name == "ldap_error_realm_is_none" %}
                         {% trans %}
                         You are trying to login using LDAP without creating an
                         organization first. Please use EmailAuthBackend to create
                         your organization and then try again.
                         {% endtrans %}
                     {% endif %}
 
-                    {% if smtp_error %}
+                    {% if social_backend_name == "smtp" %}
```

Comment:
I don't think it quite makes sense to have ``smtp`` here, as it doesn't really have anything with these auth backends?